dbt/include/snowflake/profile_template.yml

```diff
@@ -2,33 +2,33 @@ fixed:
   type: snowflake
 prompts:
   account:
-    hint: 'https://<this_value>.snowflakecomputing.com'
+    hint: 'What is your account id for Snowflake? [https://<this_value>.snowflakecomputing.com]'
   user:
-    hint: 'dev username'
+    hint: 'What is your personal user login for Snowflake?'
```

Comment:
I feel like we should either go with `personal user` here and `personal password` below, or just `user` here and `password` below.

Reply:
Resolved via https://github.com/dbt-labs/dbt-snowflake/pull/729/commits/58ce8f2838a9f9b91c11c2cb3d92aa568c09d25a